ocaml/xapi/extauth_plugin_ADwinbind.ml

```diff
@@ -221,6 +196,64 @@ let clean_local_resources () : unit =
     error "%s : %s" msg (ExnHelper.string_of_exn e) ;
     raise (Auth_service_error (E_GENERIC, msg))
 
+module Winbind = struct
+  let name = "winbind"
+
+  let is_ad_enabled ~__context =
+    ( Helpers.get_localhost ~__context |> fun self ->
+      Db.Host.get_external_auth_type ~__context ~self )
+    |> fun x -> x = Xapi_globs.auth_type_AD
```

Comment:
Is the `fun x -> x =` redundant? Just use `= Xapi_globs.auth_type_AD`

Reply:
Compiler does not happy with the new style,  so I just keep it.